test/markdownTest.js

```diff
@@ -99,30 +99,34 @@ describe('markdown', () => {
   })
 
   describe('block quote', () => {
-    it('should leave it alone if the block quote delimiter is preceded by non-whitespace content', () => {
```

Comment:
There must've been a reason to do this, any ide why this was implemented this way?

Reply:
@samatclearfeed can you post a screenshot of how the UI looks for a sentence like this:

`4 > 3`

Reply:
<img width="661" alt="Screenshot 2024-09-23 at 10 11 55 AM" src="https://github.com/user-attachments/assets/e077bb20-3508-48fe-9f99-d9e10a157f78">

Reply:
yeah, this seems to be breaking. looking for a fix in this.